src/appleseed/foundation/platform/system.cpp

```diff
@@ -95,6 +99,13 @@
 
 #endif
 
+#ifdef APPLESEED_WITH_GPU
```

Comment:
In the block above, you haven't added blank lines:
```
#ifdef APPLESEED_WITH_GPU
#include "foundation/cuda/cudadevice.h"
#include "foundation/cuda/exception.h"
#endif
```

You should do the same here.

Reply:
I prefer to be consistent with the per OS / system blocks, just because they are closer.